hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/audit/TestOzoneAuditLogger.java

```diff
@@ -143,7 +143,31 @@ public void notLogReadEvents() throws IOException {
     verifyNoLog();
   }
 
-  private void verifyLog(String expected) throws IOException {
+  /**
+   * Test to verify if multiline entries can be checked.
+   */
+
+  @Test
+  public void messageIncludesMultilineException() throws IOException {
+    String exceptionMessage = "Dummuy exception message";
```

Comment:
there is a typo here in the string Dummuy ;)

Reply:
fixed